tests/storage.spec.ts

```diff
@@ -18,7 +18,20 @@ import { activateOverride, setInput } from "./helpers/configuration";
 import { TIMEOUT } from "./helpers/constants";
 import { randomSnapshotName } from "./helpers/snapshots";
 
-test("storage pool create, edit and remove", async ({ page }) => {
+let volume = randomVolumeName();
+let page: Page;
+test.beforeAll(async ({ browser, browserName }) => {
+  page = await browser.newPage();
+  volume = `${browserName}-${volume}`;
+  await createVolume(page, volume);
+});
+
+test.afterAll(async () => {
+  await deleteVolume(page, volume);
+  await page.close();
+});
+
+test("storage pool create, edit and remove", async () => {
   const pool = randomPoolName();
```

Comment:
No browser-dependent name for the storage pool?

Reply:
since the pool creation is scoped to a single test, we don't need browser specific name for it. The browser specific naming is really a safety net for parallel tests potentially editing the same resource causing a race condition, in this case that won't happen